### PR TITLE
framework/wifi_manager: Arrange DHCPS in WiFi Manager

### PR DESCRIPTION
**Pull request by SeonghoByeon**
- Add a supplement logic for handling a number of associated stations
- Clean up the linked headers

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [0cedbf24aa7638eb4009a3301cb9bce4be680c07] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0cedbf24aa7638eb4009a3301cb9bce4be680c07] - Code Rule Check OK.

File: framework/src/wifi_manager/wifi_manager_dhcp_internal.c [hunk 1]
```diff
@@ -57,18 +63,27 @@ static uint8_t num_sta = 0;
...
+	}
+	nvdbg("increase num_sta by 1\n");
 	num_sta++;
 }
 
 void dhcps_dec_num(void)
 {
-	if (num_sta > 0) {
-		num_sta--;
+	if (num_sta <= 0) {
```
Comment (Taejun-Kwon) [inline framework/src/wifi_manager/wifi_manager_dhcp_internal.c]: it is uint8_t, it never lower than 0

**Comment by seinfra** [comment]
Target : [94421fd55d3567268b6076556c9c3412bf71b21a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [94421fd55d3567268b6076556c9c3412bf71b21a] - Code Rule Check OK.

File: framework/src/wifi_manager/wifi_manager_dhcp_internal.c (hunk 1 above)
Comment (SeonghoByeon) [reply framework/src/wifi_manager/wifi_manager_dhcp_internal.c]: Applied. Thanks!

**Pull request merged**